components/Starknet/modules/ROOT/pages/index.adoc

```diff
@@ -65,12 +65,11 @@ specialized programming language.
 
 == Contribute to the Starknet docs
 
-If you are interested in contributing to Starknet technical documentation, the following list provides quick links to help you get started:
+Want to contribute to Starknet documentation? Get started here:
 
-[horizontal,labelwidth=20,itemwidth=80]
-How to contribute:: For information on how you can contribute, see link:https://github.com/starknet-io/starknet-docs/blob/dev/README.adoc#different_ways_to_contribute[Different ways to contribute].
-Guidelines:: For basic guidelines to help us keep our content consistent, see link:https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/doc_guidelines.adoc[Documentation guidelines].
-Style guide:: See the link:https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/starknet_docs_style_guide.adoc[Starknet documentation supplementary style guide].
+* See the link:https://github.com/starknet-io/starknet-docs/blob/dev/README.adoc#different_ways_to_contribute[different ways to contribute].
+* Review these link:https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/doc_guidelines.adoc[basic writing guidelines] help you write better and be part of the team.
+* Use the link:https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/starknet_docs_style_guide.adoc[The Starknet documentation supplementary style guide]. It includes style guidance to help you write more efficiently and consistently using the Starknet voice.
```

Comment:
```suggestion
* Use the link:https://github.com/starknet-io/starknet-docs/blob/dev/contributing_to_docs/starknet_docs_style_guide.adoc[Starknet documentation supplementary style guide]. It can help you write more efficiently and consistently using the Starknet voice.
```